### ISSUE TITLE
Remaps do not work when using a non-Default profile.

### ISSUE THREAD

**Issue by zjtalia** [VSCodeVim/Vim#8520]
**Describe the bug**
Remap settings are not read from `settings.json` when using a non-Default profile.

**To Reproduce**

1.  Click `Settings` gear in bottom left corner
2.  Hover on `Profiles`, click `Create Profile...`
3.  Select `Using Current Profile` and provide a profile name.
4.  Make sure `settings.json` is included in profile creation, click `Create Profile`
5.  Reload when prompted, and open `settings.json`
6.  Settings for vim remaps are greyed out and no longer work.

**Expected behavior**
Remaps should continue to work when using a non-Default profile.

**Screenshots**
Path to file is `/c:/Users/<username>/AppData/Roaming/Code/User/profiles/257436e3/settings.json`
Other settings are still used but not remaps.
![image](https://github.com/VSCodeVim/Vim/assets/97980199/cdcd2243-8a56-4851-98cb-42f4d7fa3092)

**Environment (please complete the following information):**

- Extension (VsCodeVim) version: v1.25.2
- VSCode version: 1.80.2
- OS: Windows

**Additional context**

**Comment by ruslanSorokin** [comment]
Same here. I my case they are not greyed as unusable or unknown, but they just don't work. 
**Environment** 
* Extension (vscodevim.vim) version: v1.25.2
* VSCode version: 1.81.1 (user setup)
* OS: Windows with WSL2
![image](https://github.com/VSCodeVim/Vim/assets/88016602/2faabe2b-ca76-4863-9205-fbb249e3805c)

**Comment by dreaming12** [comment]
![image](https://github.com/VSCodeVim/Vim/assets/57713800/c96186d0-c0ab-435c-8f7e-f5873c845e5e)

![image](https://github.com/VSCodeVim/Vim/assets/57713800/d7ca94d3-d244-42b8-8358-251a1471518e)

This works for me, some configs (you can see them in extension details) must be put in the settings file: `C:/Users/<username>/AppData/Roaming/Code/User/settings.json`, not those under `profiles/` (this extension may need to improve). For other configs, just put them in your workspace setting file.

But I only copy-paste some basic config from the extension details.

Version: 1.81.1 (system setup)
Commit: 6c3e3dba23e8fadc360aed75ce363ba185c49794
Date: 2023-08-09T22:22:42.175Z
Electron: 22.3.18
ElectronBuildId: 22689846
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.22000
vim extension: v1.25.2
WSL2 version: 1.2.5.0

**Comment by polo5** [comment]
Yeah I can confirm that my vim settings are greyed out when using a non-default Profile, showing the error "This setting has an application scope and can only be set in user settings". 

My imperfect fix was to switch to default profile (bottom left icon) and copy paste all my custom vim keybindings in the default settings.json.

**Comment by doei** [comment]
same problem here. The remaps are greyed-out with the message "This setting has an application scope and can only be set in user settings". They do work up until I go into debug mode, then they're completely gone and I have to reload vscode for them to activate again.

**Comment by aguynamedben** [comment]
+1, same thing here
